modin/pandas/test/internals/test_benchmark_mode.py

```diff
@@ -15,17 +15,38 @@
 
 from contextlib import nullcontext
 import modin.pandas as pd
-from modin.pandas.test.utils import test_data_values
+from modin.pandas.test.utils import test_data_values, df_equals
 from modin.config import BenchmarkMode, StorageFormat
 from modin.test.test_utils import warns_that_defaulting_to_pandas
 
 
+default_to_pandas_context = (
+    warns_that_defaulting_to_pandas()
+    if StorageFormat.get() == "Omnisci"
+    else nullcontext()
+)
+
+
 def test_syncronous_mode():
     assert BenchmarkMode.get()
     # On Omnisci storage, transpose() defaults to Pandas.
-    with (
-        warns_that_defaulting_to_pandas()
-        if StorageFormat.get() == "Omnisci"
-        else nullcontext()
-    ):
+    with default_to_pandas_context:
         pd.DataFrame(test_data_values[0]).mean()
+
+
+def test_serialization():
+    assert BenchmarkMode.get()
+
+    sr1 = pd.Series(range(10))
+    # On Omnisci `__finalize__` raises a warning
+    with default_to_pandas_context:
+        constructor, args = sr1.__reduce__()
+    sr2 = constructor(*args)
```

Comment:
It's a bit non-obvious, why would need to use these methods?

Reply:
it's `__reduce__` signature, it returns a tuple of a construction function and an arguments to pass in it in order to deserialize the object
https://github.com/modin-project/modin/blob/8adf28f62c09d49ebb677cf14ad54a3361329b22/modin/pandas/series.py#L2516-L2520